lib/frontend/components/discover/event_card.dart

```diff
@@ -22,64 +23,72 @@ class SmallEventCard extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ClipRRect(
-      borderRadius: BorderRadius.all(
-        Radius.circular(10.0),
-      ),
-      child: DecoratedBox(
-        decoration: BoxDecoration(
-          color: AppColor.dp3,
+    return GestureDetector(
+      onTap: () {
+        Navigator.push(context, MaterialPageRoute(
```

Comment:
Use named routes when pushing new pages to remain consistent with the rest of the application.